st2actions/conf/logging.conf

```diff
@@ -36,7 +36,7 @@ datefmt=
 
 [formatter_verboseConsoleFormatter]
 class=st2common.logging.formatters.ConsoleLogFormatter
-format=%(asctime)s %(thread)s %(levelname)s %(module)s [-] %(message)s
+format=%(asctime)s %(process)d %(thread)s %(levelname)s %(module)s [-] %(message)s
```

Comment:
I am ok with including process in the line number. If we maintain per pid logging will end up being redundant.